src/store/modules/history.js

```diff
@@ -28,6 +31,9 @@ export const state = initialState()
 
 export const getters = {
   get: (state, getters, rootState, rootGetters) => id => {
+    if (!state.entries[id]) {
+      return undefined
```

Comment:
can just be `return` (is implicitly `undefined`) and seems reasonable to make into a oneliner...

Reply:
Yes, I'll get rid of it